cmd/flags_header.go

```diff
@@ -59,3 +50,61 @@ func ParseHeadersFlags(cmd *cobra.Command, env *Env) error {
 
 	return nil
 }
+
+// TrustedPeerFlags gives a set of hardcoded Header package TrustedPeer flags.
+func TrustedPeerFlags() *flag.FlagSet {
+	flags := &flag.FlagSet{}
+
+	flags.String(
+		headersTrustedPeerFlag,
+		"",
+		"Multiaddr of a reliable peer to fetch headers from. (Format: multiformats.io/multiaddr)",
+	)
+
+	return flags
+}
```

Comment:
I don't understand why this needs to be a function?

Maybe I'm missing something? cc @Wondertan

Reply:
I was just following the convention that already existed. I briefly considered trying to get [Viper](https://github.com/spf13/viper) following the conventions [here](https://github.com/carolynvs/stingoftheviper) but it took me a bit to wrap my head around the whole config process.

Reply:
Yeah I don't see the purpose in making this a function rather than just a var.

Reply:
We can't make it a var because then multiple commands will use the same instance of one global flag. Remember that we have multiple node types(bridge and light) and for each of them own flag should be created.

Reply:
ack.